repair/row_level.cc

```diff
@@ -1097,11 +1097,25 @@ class repair_meta {
         });
     }
 
+    future<size_t> get_repair_rows_external_memory_usage(const std::list<repair_row>& rows) const {
```

Comment:
With get_repair_rows_external_memory_usage(), we are having two versions of the function to get the size of the row buffer size. It is hard to reason about when we should use the new one and the old one. I think we use the new version for everything. And we can keep using the old repair_row.size() for that.